python/pyspark/pandas/series.py

```diff
@@ -6322,13 +6322,21 @@ def argmax(self, axis: Axis = None, skipna: bool = True) -> int:
             # If the maximum is achieved in multiple locations, the first row position is returned.
             return -1 if max_value[0] is None else max_value[1]
 
-    def argmin(self) -> int:
+    def argmin(self, axis: Axis = None, skipna: bool = True) -> int:
         """
         Return int position of the smallest value in the Series.
 
         If the minimum is achieved in multiple locations,
         the first row position is returned.
 
+        Parameters
+        ----------
+        axis : {{None}}
+            Dummy argument for consistency with Series.
+        skipna : bool, default True
+            Exclude NA/null values. If the entire Series is NA, the result
```

Comment:
> If the entire Series is NA, the result

It seems a plus doc for pandas (1.4.3), do we want to add a test for this?

[1] https://pandas.pydata.org/docs/reference/api/pandas.Series.argmin.html

Reply:
it results -1, not NA.

this behavior is the same as Pandas.

good catch!